projects/canopy/src/lib/filter-buttons/filter-button/filter-button.component.html

```diff
@@ -0,0 +1,21 @@
+<input
+  (click)="onCheck()"
+  [attr.aria-describedby]="variant === 'select-multiple' ? ariaDescribedBy : null"
+  [attr.checked]="checked === true ? true : null"
```

Comment:
Could this be `[attr.checked]="checked || null"`?

Reply:
Done.